### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Arashfa0301**
- setup a next react project that uses bun 
- setup a node express project for the backend

### CODE REVIEW

**Review by haukkagu — APPROVED** [review]
LGTM. Although it should not be called 'initial commit" :angry:

File: client/components/subtitle.tsx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+export default function Hey() {
```
Comment (itsisak) [inline client/components/subtitle.tsx:2]: Do we need this?

File: .DS_Store
Comment (itsisak) [inline .DS_Store:1]: please add this to `.gitignore` instead

File: client/public/vercel.svg
Comment (itsisak) [inline client/public/vercel.svg:1]: I dont think this is necessary

**Review by itsisak — APPROVED** [review]
Looks good, but a few unnecessary things

**Pull request merged**